### DISCUSSION TITLE
Top Nav Menu in CardView

### DISCUSSION THREAD

**Discussion by davidjhurtado** [Megabit/Blazorise#4630] [Q&A]
Is possible or there is a way to create a Top Nav menu on a CardView Control?
We have a Dashboard with distincts charts, and we need users could change filter and other conditions, we need to provide to the users a rapid way to set this conditions
We use abp commercial framework with version 1.2.0 of Blazorise.

**Comment by stsrki** [comment]
We would like to help you, but we're not familiar with the CardView component. It is best is could ask about it on ABP project repo.

**Reply by davidjhurtado** [reply]
Sorry i did not explain well 
I am refering to a Blazorise Card component. 
"Blazorise Card component"= > [https://blazorise.com/docs/components/card](https://blazorise.com/docs/components/card)
We need to make something like this
![image](https://user-images.githubusercontent.com/13089183/221563117-f6cbec56-66af-4588-9343-d8cbf7ce7f2d.png)
